### PR TITLE
Display token amount with metric suffix

### PR DESCRIPTION
**Pull request by r-czajkowski**
Depends on: #1721 

obraz

### CODE REVIEW

File: solidity/dashboard/src/utils/token.utils.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+ */
+export function fromTokenUnit(amount, decimals = 18) {
+  amount = new BigNumber(amount)
+  return amount.times(new BigNumber(10).pow(new BigNumber(decimals)))
+}
+
+/**
+ * Returns a number with a metric suffix eg.:
+ * * 10000000 => { value: 10, suffix: 'M', formattedValue: '10M' }
+ * * 1000.4 => { value: 1.4, suffix: 'K', formattedValue: '10K'}
```
Comment (dimpar) [inline solidity/dashboard/src/utils/token.utils.js:47]: shouldn't it be 1.4K ?

File: solidity/dashboard/src/utils/token.utils.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+ * * 10000.4 => { value: 10, suffix: 'K', formattedValue: '10K' }
```
Comment (dimpar) [inline solidity/dashboard/src/utils/token.utils.js:48]: 10.4K ?

File: solidity/dashboard/src/components/OwnedTokensOverview.jsx [hunk 3]
```diff
@@ -33,7 +38,7 @@ const OwnedTokensOverview = ({ keepBalance, stakedBalance }) => {
           <h5 className="text-grey-70">staked</h5>
           <h4 className="text-grey-70">{displayAmount(stakedBalance)}</h4>
```
Comment (dimpar) [inline solidity/dashboard/src/components/OwnedTokensOverview.jsx:39]: do we want to display formatted amount here as well?

File: solidity/dashboard/src/utils/token.utils.js (hunk 2 above)
Comment (r-czajkowski) [reply solidity/dashboard/src/utils/token.utils.js:48]: We want to display decimals only if a value is less than 10.

File: solidity/dashboard/src/components/OwnedTokensOverview.jsx (hunk 3 above)
Comment (r-czajkowski) [reply solidity/dashboard/src/components/OwnedTokensOverview.jsx:39]: I did it according to Figma.

**Pull request merged**